pkgs/development/python-modules/comicon/default.nix

```
@@ -14,34 +13,23 @@
 
 buildPythonPackage rec {
   pname = "comicon";
-  version = "1.3.0";
+  version = "1.4.0";
   pyproject = true;
   disabled = pythonOlder "3.10";
 
   src = fetchFromGitHub {
     owner = "potatoeggy";
     repo = "comicon";
     tag = "v${version}";
-    hash = "sha256-0AGCTnStyBVL7DVkrUFyD60xnuuO1dcl+Twdyy+uq1Y=";
+    hash = "sha256-jZ/ZhSFg0ZPTYI10s8Cn1s9UZRFFnuLjS96lnUFVekQ=";
   };
 
-  patches = [
-    # Upstream forgot to bump the version before tagging
-    # See https://github.com/potatoeggy/comicon/commit/d698f0f03b1a391f988176885686e9fca135676e
-    (fetchpatch2 {
-      name = "comicon-version-bump.patch";
-      url = "https://github.com/potatoeggy/comicon/commit/d698f0f03b1a391f988176885686e9fca135676e.diff";
-      hash = "sha256-ZHltw4OSYuHF8mH0kBZDsuozPy08Bm7nme+XSwfGNn8=";
-    })
-  ];
-
   nativeBuildInputs = [
```

```suggestion
  build-system = [
```

and `dependencies` as well, for both package, please.